Bigtable/src/ChunkFormatter.php

```diff
@@ -250,9 +250,11 @@ private function updateReadRowsRequest($request, $options, $prevRowKey)
                     if (($range->getEndKeyOpen() && $prevRowKey > $range->getEndKeyOpen())
                         || ($range->getEndKeyClosed() && $prevRowKey >= $range->getEndKeyClosed())) {
                         continue;
-                    } elseif ((!$range->getStartKeyOpen() || $prevRowKey > $range->getStartKeyOpen())
+                    }
+                    if ((!$range->getStartKeyOpen() || $prevRowKey > $range->getStartKeyOpen())
                         && (!$range->getStartKeyClosed() || $prevRowKey >= $range->getStartKeyClosed())) {
-                        $range->setStartKeyOpen($prevRowKey);
+                        $rowKeySetter = $request->getReversed() ? 'setEndKeyOpen' : 'setStartKeyOpen';
```

Comment:
This feature (supporting reversed row keys) is not supported in PHP (and is not prioritized), but is required for the conformance tests to pass.

I don't like adding support for it here, but without it we would need to skip the `LastScannedRow` test

Reply:
I don't think we need to add reverse scan support, reverse scan conformance test should be separate from the regular scans?

Reply:
I 100% agree, but unfortunately this test is `TestReadRows_Retry_LastScannedRow`, and it's required for that test to pass. I don't know why it's setting `reversed` in the conformance tests... maybe this is something we can have fixed there? It could also be because I'm using `v0.0.3`.

Reply:
Ahh okay, it does look like this was split into a separate test in the [main branch](https://github.com/googleapis/cloud-bigtable-clients-test/blob/7597087db6059e41cf922637464b9e309b7db445/tests/readrows_test.go#L308). However, we need a new release tagged, as I don't want to pin these tests to `main`. They're _currently broken_ for this reason (an update to `main` has broken them).

Could. you cut a release for `v0.0.4`?

Reply:
Nevermind, I was wrong! There was always a `TestReadRows_Retry_LastScannedRow_Reverse` and I just missed it.

I added it to `known_failures.txt` and removed all `Reverse` logic, and everything is passing as expected.